--- unity-5.12.0/standalone-clients/CMakeLists.txt.orig	2012-05-19 14:38:48.151559359 +0100
+++ unity-5.12.0/standalone-clients/CMakeLists.txt	2012-05-19 14:39:07.875560336 +0100
@@ -202,120 +202,6 @@
                 )
 add_dependencies (switcher unity-core-${UNITY_API_VERSION})
 
-add_executable (launcher
-                TestLauncher.cpp
-                ${UNITY_SRC}/CairoBaseWindow.cpp
-                ${UNITY_SRC}/AbstractLauncherIcon.h
-                ${UNITY_SRC}/AbstractLauncherIcon.cpp
-                ${UNITY_SRC}/AbstractIconRenderer.h
-                ${UNITY_SRC}/LauncherIcon.cpp
-                ${UNITY_SRC}/LauncherIcon.h
-                ${UNITY_SRC}/SimpleLauncherIcon.cpp
-                ${UNITY_SRC}/SimpleLauncherIcon.h
-                ${UNITY_SRC}/BamfLauncherIcon.cpp
-                ${UNITY_SRC}/BamfLauncherIcon.h
-                ${UNITY_SRC}/BFBLauncherIcon.cpp
-                ${UNITY_SRC}/BFBLauncherIcon.h
-                ${UNITY_SRC}/SpacerLauncherIcon.cpp
-                ${UNITY_SRC}/SpacerLauncherIcon.h
-                ${UNITY_SRC}/DndData.cpp
-                ${UNITY_SRC}/DndData.h
-                ${UNITY_SRC}/TrashLauncherIcon.cpp
-                ${UNITY_SRC}/TrashLauncherIcon.h
-                ${UNITY_SRC}/Decaymulator.cpp
-                ${UNITY_SRC}/Decaymulator.h
-                ${UNITY_SRC}/DesktopLauncherIcon.cpp
-                ${UNITY_SRC}/DesktopLauncherIcon.h
-                ${UNITY_SRC}/DNDCollectionWindow.cpp
-                ${UNITY_SRC}/DNDCollectionWindow.h
-                ${UNITY_SRC}/DeviceLauncherIcon.cpp
-                ${UNITY_SRC}/DeviceLauncherIcon.h
-                ${UNITY_SRC}/DeviceLauncherSection.cpp
-                ${UNITY_SRC}/DeviceLauncherSection.h
-                ${UNITY_SRC}/DevicesSettings.cpp
-                ${UNITY_SRC}/DevicesSettings.h
-                ${UNITY_SRC}/EdgeBarrierController.h
-                ${UNITY_SRC}/EdgeBarrierController.cpp
-                ${UNITY_SRC}/FavoriteStore.cpp
-                ${UNITY_SRC}/FavoriteStore.h
-                ${UNITY_SRC}/FavoriteStoreGSettings.cpp
-                ${UNITY_SRC}/FavoriteStoreGSettings.h
-                ${UNITY_SRC}/FavoriteStorePrivate.cpp
-                ${UNITY_SRC}/FavoriteStorePrivate.h
-                ${UNITY_SRC}/HudLauncherIcon.cpp
-                ${UNITY_SRC}/HudLauncherIcon.h
-                ${UNITY_SRC}/IconLoader.cpp
-                ${UNITY_SRC}/IconLoader.h
-                ${UNITY_SRC}/IconTextureSource.h
-                ${UNITY_SRC}/IconTextureSource.cpp
-                ${UNITY_SRC}/LauncherOptions.cpp
-                ${UNITY_SRC}/LauncherOptions.h
-                ${UNITY_SRC}/LauncherEntryRemoteModel.cpp
-                ${UNITY_SRC}/LauncherEntryRemoteModel.h
-                ${UNITY_SRC}/LauncherEntryRemote.cpp
-                ${UNITY_SRC}/LauncherEntryRemote.h
-                ${UNITY_SRC}/LauncherController.cpp
-                ${UNITY_SRC}/LauncherController.h
-                ${UNITY_SRC}/LauncherModel.cpp
-                ${UNITY_SRC}/LauncherModel.h
-                ${UNITY_SRC}/Launcher.cpp
-                ${UNITY_SRC}/Launcher.h
-                ${UNITY_SRC}/LauncherHideMachine.cpp
-                ${UNITY_SRC}/LauncherHideMachine.h
-                ${UNITY_SRC}/LauncherHoverMachine.cpp
-                ${UNITY_SRC}/LauncherHoverMachine.h
-                ${UNITY_SRC}/LauncherDragWindow.cpp
-                ${UNITY_SRC}/LauncherDragWindow.h
-                ${UNITY_SRC}/GeisAdapter.cpp
-                ${UNITY_SRC}/GeisAdapter.h
-                ${UNITY_SRC}/WindowManager.h
-                ${UNITY_SRC}/WindowManager.cpp
-                ${UNITY_SRC}/IconRenderer.cpp
-                ${UNITY_SRC}/IconRenderer.h
-                ${UNITY_SRC}/Tooltip.cpp
-                ${UNITY_SRC}/Tooltip.h
-                ${UNITY_SRC}/BackgroundEffectHelper.h
-                ${UNITY_SRC}/BackgroundEffectHelper.cpp
-                ${UNITY_SRC}/StaticCairoText.cpp
-                ${UNITY_SRC}/StaticCairoText.h
-                ${UNITY_SRC}/SingleMonitorLauncherIcon.cpp
-                ${UNITY_SRC}/SingleMonitorLauncherIcon.h
-                ${UNITY_SRC}/SoftwareCenterLauncherIcon.cpp
-                ${UNITY_SRC}/SoftwareCenterLauncherIcon.h
-                ${UNITY_SRC}/Introspectable.cpp
-                ${UNITY_SRC}/Introspectable.h
-                ${UNITY_SRC}/PanelStyle.cpp
-                ${UNITY_SRC}/PanelStyle.h
-                ${UNITY_SRC}/PointerBarrier.cpp
-                ${UNITY_SRC}/PointerBarrier.h
-                ${UNITY_SRC}/QuicklistMenuItem.cpp
-                ${UNITY_SRC}/QuicklistMenuItem.h
-                ${UNITY_SRC}/QuicklistMenuItemCheckmark.cpp
-                ${UNITY_SRC}/QuicklistMenuItemCheckmark.h
-                ${UNITY_SRC}/QuicklistMenuItemLabel.cpp
-                ${UNITY_SRC}/QuicklistMenuItemLabel.h
-                ${UNITY_SRC}/QuicklistMenuItemRadio.cpp
-                ${UNITY_SRC}/QuicklistMenuItemRadio.h
-                ${UNITY_SRC}/QuicklistMenuItemSeparator.cpp
-                ${UNITY_SRC}/QuicklistMenuItemSeparator.h
-                ${UNITY_SRC}/QuicklistView.cpp
-                ${UNITY_SRC}/QuicklistView.h
-                ${UNITY_SRC}/QuicklistManager.cpp
-                ${UNITY_SRC}/QuicklistManager.h
-                ${UNITY_SRC}/TextureCache.cpp
-                ${UNITY_SRC}/TextureCache.h
-                ${UNITY_SRC}/Timer.cpp
-                ${UNITY_SRC}/Timer.h
-                ${UNITY_SRC}/UBusMessages.h
-                ${UNITY_SRC}/UBusWrapper.cpp
-                ${UNITY_SRC}/UBusWrapper.h
-                ${UNITY_SRC}/ubus-server.cpp
-                ${UNITY_SRC}/ubus-server.h
-                ${UNITY_SRC}/UScreen.cpp
-                ${UNITY_SRC}/UScreen.h
-                )
-add_dependencies (launcher unity-core-${UNITY_API_VERSION})
-
 add_executable (keyutil
                 TestKeyboardUtil.cpp
                 ${UNITY_SRC}/KeyboardUtil.h
@@ -567,7 +453,7 @@
 add_dependencies (screen-effect-fbo ${UNITY_API_VERSION})
 
 # Custom target to make all the other targets here, add your test to this list
-add_custom_target(standalone-clients DEPENDS dash panel launcher switcher keyutil quicklist quicklist-visuals filters filter-bar preview-applicaiton preview-generic preview-music result-view dash-style bg-hash shortcut-view tooltip)
+add_custom_target(standalone-clients DEPENDS dash panel switcher keyutil quicklist quicklist-visuals filters filter-bar preview-applicaiton preview-generic preview-music result-view dash-style bg-hash shortcut-view tooltip)
 
 
 
